portal/src/texts/profile/icon.mdx

```diff
@@ -19,8 +23,11 @@ Ikonene vi bruker er gjerne en del av et skjemafelt, for eksempel [Select](/komp
 
 ## Ikonene våre
 
-<PortalIconExample />
+<ComponentExample title="Icons" component={IconsExample} knobs={iconsExampleKnobs} />
```

Comment:
Her funderte vi vel om det heller er hensiktsmessig å linke til komponentsiden for ikonene. De ser forsåvidt OK ut _as is_ da, så sikkert ingen fare.

Reply:
Vi snakket litt om det i går, og fant ut at vi kanskje skulle røske litt ordentlig i de profilsidene som en egen oppgave, kanskje til og med i portal 3.0